code/modules/language/foxspeak.dm

```diff
@@ -0,0 +1,33 @@
+/datum/language/canilunzt
+	name = "Canilunzt"
+	desc = "The common language of foxpeople, composed of low growls and yelps."
+	speech_verb = "growls"
+	ask_verb = "growls"
+	exclaim_verb = "roars"
+	key = "7"
+	flags = TONGUELESS_SPEECH
+	space_chance = 20
+	syllables = list("rur","ya","cen","rawr","bar","kuk","tek","qat","uk","wu","vuh","tah","tch","schz","auch",
+	"ist","ein","entch","zwichs","tut","mir","wo","bis","es","vor","nic","gro","lll","enem","zandt","tzch","noch",
+	"hel","ischt","far","wa","baram","iereng","tech","lach","sam","mak","lich","gen","or","ag","eck","gec","stag","onn",
+	"bin","ket","jarl","vulf","einech","cresthz","azunein","ghzth")
+	icon_state = "fox"
+	default_priority = 90
+
+/proc/generate_random_names(amount = 500)
```

Comment:
I have yet to actually find a use for this proc. I'll need more coffee and some angst before I sit down to create a namelist. This proc will be removed; not sure why I left it in here.